log.capnp

```diff
@@ -1614,7 +1614,36 @@ struct Joystick {
   buttons @1: List(Bool);
 }
 
-struct DriverState {
+struct DriverStateV2 {
+  frameId @0 :UInt32;
+  modelExecutionTime @1 :Float32;
+  dspExecutionTime @2 :Float32;
+  rawPredictions @3 :Data;
+
+  poorVision @4 :Float32;
+  wheelOnRight @5 :Float32;
+
+  driverDataLH @6 :DriverData;
+  driverDataRH @7 :DriverData;
+
+  struct DriverData {
```

Comment:
should we come up with a better name than driver?

Reply:
can use "occupant" instead? if that doesn't sound weird